src/main/resources/assets/identity/lang/sk_sk.json

```
@@ -0,0 +1,31 @@
+{
+  "identity.unlock_entity": "Odomknutá %s identita.",
+  "identity.revoke_entity": "Vaša pamäť bytia %s sa vyparila...",
+  "identity.grant_success": "Úspešne dodalo %s identitu pre %s.",
+  "identity.revoke_success": "Úspešne odobralo %s identitu od %s.",
+  "identity.already_has": "%s už má %s identitu!",
+  "identity.does_not_have": "%s nemá %s identitu!",
+  "identity.equip_success": "Nastavená identita %s pre %s.",
+  "identity.unequip_success": "Vymazaná identita od %s.",
+  "identity.test_positive": "%s je teraz %s.",
+  "identity.test_failed": "%s nie je %s.",
+  "identity.menu_hint": "Aby ste dostali identitu, musíte zabiť príslušnú entitu...",
+  "identity.help": "Otvoriť pomocníka Identity módu",
+  "identity.help.welcome": "Vitajte. Identity je moderná verzia Morphu pre Fabric napísaná Draylarom.",
+  "identity.help.credits": "Ďakujeme všetkým ktorí nás podporujú s návrhmi a hlásením chýb!",
+  "identity.help.support_label": "Chcete nahlásiť chybu alebo podať návrh?",
+  "identity.help.support_description": "Choďte na Identity Github https://github.com/Draylar/identity.",
+  "identity.help.ability_label": "Používanie schopností",
+  "identity.help.ability_description_1": "Aby ste použili schopnosť identity, stlačte tlačidlo R (predvolené).",
+  "identity.help.ability_description_2": "Indikátor schladenia sa ukáže vpravo-dole aby ste vedeli kedy to môžete použiť znovu.",
+  "identity.help.ability_description_3": "Pre celý zoznam schopností príďte na wiki: https://github.com/Draylar/identity/wiki/Abilities.",
+  "identity.help.config_label": "Nastavujem Identity",
+  "identity.help.config_description": "Choďte do súboru nastavení: /config/identity.json5 pre všetky nastavenia módu.",
+  "identity.help.credits_label": "Titulky",
+  "identity.help.credits_general": "Všeobecné: Pyrofab, AllanChain, James103, lawleagle",
+  "identity.help.credits_translators": "Prekladatelia: qsefthuopq, Godzmoline, SmajloSlovakian", //I added my name in here in case of this translation being added ;)
```

Comments are not supported in JSON.
```suggestion
  "identity.help.credits_translators": "Prekladatelia: qsefthuopq, Godzmoline, SmajloSlovakian",
```